sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -142,6 +140,24 @@ def __init__(self, name, init_config, instances):
             ttl=60 * 60 * 24,
         )
 
+        # Query declarations
+        self.server_state_queries = QueryExecutor(
```

Comment:
Do you anticipate these queries being costly? I'm wondering if they should be behind a config option instead of always turned on.

Reply:
Good to know, thanks!